_1_basics/src/main/java/code/_4_student_effort/FizzBuzz/Main.java

```diff
@@ -0,0 +1,14 @@
+package code._4_student_effort.FizzBuzz;
+
+public class Main {
+    public static void main(String[] args) {
+        FizzBuzz fb = new FizzBuzz();
+        FizzBuzz2 fb2 = new FizzBuzz2();
+        for(int i = 1; i < 101; i++) {
+            fb.printString(i);
+        }
+        for(int i = 1; i < 101; i++) {
```

Comment:
numarul 105 este divizibil si cu 3,5,7 => ar trebui sa afiseze FizzBuzzRizz 
cate conditii noi ar trebui sa adaugi in cazul asta ?